src/gesture.js

```diff
@@ -173,6 +175,35 @@ export class Gestures {
     return overserver.add(handler);
   }
 
+  /**
+   * Unsubscribes all handlers from the given gesture recognizer. Returns
+   * true if anything was done. Returns false if there were no handlers
+   * registered on the given gesture recognizer in first place.
+   *
+   * @param {function(new:GestureRecognizer<DATA>, !Gestures)} recognizerConstr
+   * @return {boolean}
+   */
+  removeGesture(recognizerConstr) {
+    const type = new recognizerConstr(this).getType();
+    const overserver = this.overservers_[type];
```

Comment:
It'd be nice if we could get the type of the constructor without creating an instance, but it doesn't look like there's another way to do it right now without refactoring, maybe adding an inherited static method on the subclasses to replace the hardcoded string in the constructor. Not necessary to fix, just thinking out loud.

Reply:
Can investigate feasibility and add a GFI task to refactor.